support-frontend/assets/pages/contributions-checkout-in-epic/ContributionsCheckoutForm.tsx

```diff
@@ -0,0 +1,266 @@
+							<div css={styles.primaryCtaContainer}>
+								{type === 'PAY_NOW' ? (
+									<Button
+										onClick={(e) => {
+											setShouldShowErrorMessage(true);
+											onCustomButtonClick(e);
+										}}
+										css={styles.customPrb}
+									>
+										Pay with saved card
+									</Button>
+								) : (
+									<PaymentRequestButtonElement
+										options={{ paymentRequest }}
+										onClick={(e) => {
+											setShouldShowErrorMessage(true);
+											onStripeButtonClick(e);
+										}}
+									/>
+								)}
+							</div>
+
+							<div css={styles.secondaryCtaContainer}>
+								<ThemeProvider theme={buttonReaderRevenue}>
+									<LinkButton
+										href={getSupportUrl(contributionType, amount)}
+										onClick={onAlternativePaymentButtonClicked}
+										cssOverrides={styles.secondaryCta}
+									>
+										<div css={styles.secondaryCtaContentsContainer}>
+											<div>
+												<PaymentIcons cssOverrides={styles.paymentIconsIn} />
+											</div>
+											<div>Pay another way</div>
+										</div>
+									</LinkButton>
+								</ThemeProvider>
+							</div>
+						</div>
+					)}
+					renderFallback={() => (
+						<div>
+							<ThemeProvider theme={buttonReaderRevenue}>
+								<LinkButton
+									href={getSupportUrl(contributionType, amount)}
+									onClick={onAlternativePaymentButtonClicked}
+									icon={<SvgArrowRightStraight />}
+									iconSide="right"
+									nudgeIcon
+									cssOverrides={styles.supportCta}
+								>
+									Support the Guardian
+								</LinkButton>
+							</ThemeProvider>
+
+							<div css={styles.paymentIconsContainer}>
+								<PaymentIcons cssOverrides={styles.paymentIcons} />
+							</div>
+						</div>
+					)}
+				/>
+			</div>
+		</div>
+	);
+}
+
+// ---- Helper components ---- //
+
+function PaymentIcons(props: { cssOverrides?: SerializedStyles }) {
+	return (
+		<img
+			width={422}
+			height={60}
+			src="https://assets.guim.co.uk/images/acquisitions/2db3a266287f452355b68d4240df8087/payment-methods.png"
+			alt="Accepted payment methods: Visa, Mastercard, American Express and PayPal"
+			css={props.cssOverrides}
+		/>
+	);
+}
+
+// ---- Helpers ---- //
+
+const BASE_URL = 'https://support.theguardian.com/contribute';
+
+function getSupportUrl(
+	contributionType: ContributionType,
+	amount: number,
+): string {
+	// The query params coming from SDC contain all of the tracking data
+	// for the A/B test. We forward all of this on to the support site
+	// in order to track the participations correctly.
+	const tracking = window.location.search.slice(1);
+
+	return `${BASE_URL}?selected-contribution-type=${contributionType}&selected-amount=${amount}&${tracking}`;
+}
+
+// ---- Styles ---- //
+
+export const styles = {
+	prbContainer: css`
+		margin-top: ${space[6]}px;
+	`,
+	ctasContainer: css`
+		display: flex;
+		flex-direction: column;
+
+		@media (min-width: 520px) {
```

Comment:
I've used specific media queries here which is pretty gross. I had to do this to accomodate the width of the CTA that contains the payment icons. More generally the media queries in this page don't actually with the breakpoints on dotcom when it's inside an epic. As this page is embedded within the epic inside of an iframe, the widths are dictated by the width of the container that holds the iframe. We effectivley get css container queries here, but that makes the source breakpoints generally less helpful

Reply:
I think it would help to have a named variable for the width that gives this context, maybe with a comment too if you feel like it's necessary, rather than just having a magic 520px in the code.

Reply:
Yeah I've done just that - thanks!